test/manual/api/system_api_tests.sh

```
@@ -0,0 +1,682 @@
+      echo "✅ Deleted connection $source->$target"
+    else
+      echo "❌ Failed to delete connection $source->$target. Status: $status"
+      [[ "$VERBOSE" == "true" ]] && echo "Response: $(parse_response "$raw")"
+    fi
+  done
+
+  echo "Cleaning up systems..."
+  # Use batch delete for systems
+  local system_array=($CREATED_SYSTEM_IDS)
+  echo "Attempting batch delete of systems..."
+  echo "System ${system_array[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)